docs/src/add-services/redis.md

```diff
@@ -189,62 +184,84 @@ const value = await client.get('x'); // returns 42
 
 {{< /codetabs >}}
 
+## Relationship
+
+After you've [configured your Redis service](#usage-example),
+you can retrieve information such as the credentials and port you need to connect to your Redis service.
+
+To do so, you can access the [`PLATFORM_RELATIONSHIPS` environment variable](../development/variables/use-variables.md#use-platformsh-provided-variables) directly [in your app](../../development/variables/use-variables.md#access-variables-in-your-app),
+for example using a Platform.sh configuration reader.
+
+The format exposed in the `PLATFORM_RELATIONSHIPS` environment variable is identical
+for [ephemeral](#ephemeral-redis) and [persistent](#persistent-redis) Redis:
```

Comment:
:hammer_and_wrench: Since you can get to it in different ways, I wouldn't put so much emphasis on the variable.
```suggestion
The format of the relationship is identical whether your Redis service is [ephemeral](#ephemeral-redis) or [persistent](#persistent-redis).
```